redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
```

Comment:
Also, sorry for being a tad informal. I didn't realize which repository I was reviewing a PR in until just now, I mistakenly thought this was a work internal thing... ;)